pennylane_lightning/core/src/utils/cuda_utils/cuGates_host.hpp

```diff
@@ -1472,6 +1472,162 @@ static constexpr auto getP1111_CU() -> std::vector<CFP_t> {
             cuUtil::ONE<CFP_t>()};
 }
 
+/**
+ * @brief Create a matrix representation of the PauliX@PauliY data in row-major
+ * format.
+ *
+ * @tparam CFP_t Required precision of gate (`float` or `double`).
+ * @return constexpr std::vector<CFP_t> Return constant expression
+ * of PauliX@PauliY data.
+ */
+template <class CFP_t> static constexpr auto getXY() -> std::vector<CFP_t> {
+    auto &&PauliY = getPauliY<CFP_t>();
+    return {PauliY[2], PauliY[3], PauliY[0], PauliY[1]};
```

Comment:
Why not return explicitly zeros?
``` c++
return {PauliY[2], 0.0, 0.0, PauliY[1]}
```

Reply:
It would be easier for maintenance since it's a direct transform from `PauliY`